inc/sainexthopgroup.h

```diff
@@ -81,6 +115,37 @@ typedef enum _sai_next_hop_group_attr_t
     SAI_NEXT_HOP_GROUP_ATTR_TYPE,
 
     /**
+     * @brief Events that can initiate switchover from primary to backup next hop
+     *
+     * Can be a combination of the following:
+     * SAI_NEXT_HOP_SWITCHOVER_TRIGGER_NULL,
+     * SAI_NEXT_HOP_SWITCHOVER_TRIGGER_BFD,
+     * SAI_NEXT_HOP_SWITCHOVER_TRIGGER_PORT_DOWN
+     *
+     * @type sai_uint8_t
```

Comment:
Hi Kamil, apologies for a late reply.  

During the call it was suggested that if control plane only indicates for example that the switching entity should trigger a switchover if the corresponding BFD session fails, then it would be non trivial for the switch to figure out which BFD session to monitor in this case.  
Therefore, we decided that it will be better if the control plane provides BFD session id rather than just a generic flag, when it wants the switch to trigger a switchover.  
The same logic applies to the port down event - control plane will provide an identifier for the associated port.

Let me know if you'd like more clarification.